### PR TITLE
fix: script call for linux

### PR DESCRIPTION
**Pull request by camilamacedo86**
**decription**
- small fix regards issue faced running the script in the Linux
- add a test  to check its execution in both platforms

Related to the PR: https://github.com/kubernetes-sigs/kubebuilder/pull/1608

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ os:
   - osx
 
 go:
-  - "1.13"
+  - 1.13.x
```
Comment (camilamacedo86) [inline .travis.yml:13]: To test we need 1.13.4 or greater

**Comment by camilamacedo86** [comment]
/test pull-kubebuilder-e2e-k8s-1-15-3

/test pull-kubebuilder-e2e-k8s-1-17-0
--

File: test.sh [hunk 2]
```diff
@@ -152,4 +152,7 @@ GO111MODULE=on test_project project-v3 3-alpha
 GO111MODULE=on test_project project-v3-multigroup 3-alpha
 GO111MODULE=on test_project project-v3-addon 3-alpha
 
+# test script that setup envtest
+./scripts/setup_envtest_bins.sh v1.18.2 v3.4.3
```
Comment (djzager) [inline test.sh:156]: I'm struggling to understand why we are installing the envtest binaries right before exiting the test.

Comment (camilamacedo86) [reply test.sh:156]: It just for we ensure that the scripts are working in both platform. 
The idea is to remove it in the follow-up https://github.com/kubernetes-sigs/kubebuilder/pull/1600 where they will be called for each testadata sample via the makefile target instead.

**Pull request merged**